### DISCUSSION TITLE
Configuration with 2 nodes

### DISCUSSION THREAD

**Discussion by azertylr** [maxpert/marmot#103] [General]
Hello,

I'd like to synchronise to vaultwarden sqlite database, so I'm trying this awesome tool! I have difficulties to understand, because between 2 local docker, it is working fine, but with 2 database, one local and one remote, with port 4222 open, I have troubles.

If I have only 2 nodes, do I need to use -cluster-addr "0.0.0.0:4222" -cluster-peers "nats://server2:4222/" or should I use, in the config file
`bind_address="0.0.0.0:4222"
urls=[ "nats://server2:4222"]`

Thank you!
Best

**Comment by maxpert** [comment]
Is the local node behind a NAT?

**Reply by azertylr** [reply]
yes, port 4222 is open
